lib/src/tabs-nav/Tab.tsx

```diff
@@ -103,16 +117,18 @@ const Tab = styled.a<TabStyleProps>`
   ${(props) =>
     !props.disabled &&
     `:hover {
-    background: #0000000D;
+    background: ${props.theme.unselectedBackgroundColor};
   }
 
   :focus {
-    outline: 2px solid #33aaff};
+    outline: 2px solid ${props.theme.focusOutline};
   }
 
   :active {
-    background: #0000001A;
+    background: ${props.theme.pressedBackgroundColor};
     outline: 2px solid #33aaff};
+    color: ${props.theme.selectedFontColor};
+
```

Comment:
This `color` sentence is not needed, since it is applied in the `DxcTypography`.

Reply:
done